### DISCUSSION TITLE
bulk rename with regex

### DISCUSSION THREAD

**Discussion by susnick** [nushell/nushell#10765] [Show and tell]
I was trying to find a way to rename torrent files. Any improvements appreciated.
`
ls | get name | into string | each {|e| $e | parse -r "(S\\d+E\\d+)" | get capture0 | each {|n| mv $e $"($n).mkv"}}
`

**Comment by amtoine** [comment]
@susnick 
i have written this [`file bulk-rename` command](https://github.com/nushell/nu_scripts/pull/643) to the standard library candidates of `nu_scripts` yesterday :smirk:

maybe that solves your issue? :innocent:
